HW13/requirements.txt

```diff
@@ -0,0 +1 @@
+webdriver-manager
```

Comment:
I don't see selenium and pytest in requirements, but you are using them in your code